frontend/src/components/campAdmin/CampCategories.vue

```diff
@@ -3,109 +3,42 @@
     <slot name="title">
       <div class="ec-content-group__title py-1 subtitle-1">
         {{ $tc('components.campAdmin.campCategories.title') }}
-        <dialog-category-create v-if="!disabled" :camp="camp()">
+        <DialogCategoryCreate v-if="!disabled" :camp="camp()">
           <template #activator="{ on }">
-            <button-add
+            <ButtonAdd
               color="secondary"
               text
               :hide-label="$vuetify.breakpoint.xsOnly"
               class="my-n2"
               v-on="on"
             >
               {{ $tc('components.campAdmin.campCategories.create') }}
-            </button-add>
+            </ButtonAdd>
           </template>
-        </dialog-category-create>
+        </DialogCategoryCreate>
       </div>
     </slot>
     <v-skeleton-loader v-if="camp().categories()._meta.loading" type="article" />
-    <v-list>
+    <v-list class="mx-n2">
       <v-list-item
         v-for="category in categories.items"
         :key="category._meta.self"
-        class="px-0"
+        class="px-2 rounded"
+        :to="categoryRoute(camp(), category)"
```

Comment:
This allows a guest to view the category edit mode.
There the buttons to add contentnodes, the drag/drop for the contentnodes and the contentnode edit functionality is not disabled, but the api requests fail.

I would make sure that the buttons are hidden, drag/drop is disabled and text cannot be edited.
![grafik](https://github.com/ecamp/ecamp3/assets/1506818/49aaa76d-ff45-4530-ae7c-3f62749cf431)
![grafik](https://github.com/ecamp/ecamp3/assets/1506818/273f7f18-5aba-4eb8-87a8-a584f8b11f9b)

Reply:
Good catch, I renamed the prop from readonly to disabled and forgot to rename it in the CategoryTemplate. Fixed in 
86a8011